From d0adc17dfc51f406ed7b17807ea8a7a555447197 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Janne=20Sir=C3=A9n?=
 <48908460+JanneSiren@users.noreply.github.com>
Date: Sun, 7 May 2023 03:18:55 +0300
Subject: [PATCH] hwdb: add landscape IdeaPad Miix 310 sensor orientation
 (#27555)

Enables support for landscape orientated display variants of the IdeaPad Miix 310 2-in-1 laptop.

(cherry picked from commit 78e0dd224467dc627dc0d3f907954e2a6dc59bc8)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index ebf8c718a7..f971f701cc 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -639,6 +639,10 @@ sensor:modalias:acpi:KIOX000A*:dmi:bvnLENOVO:bvr1HCN4?WW:*:svnLENOVO:pn80SG:*
 sensor:modalias:acpi:KIOX000A*:dmi:bvnLENOVO:bvr1HCN2?WW:*:svnLENOVO:pn80SG:*
  ACCEL_MOUNT_MATRIX=0, 1, 0; 1, 0, 0; 0, 0, 1
 
+# IdeaPad Miix 310 BIOS version bvr1HCN3?WW (variant 3)
+sensor:modalias:acpi:KIOX000A*:dmi:bvnLENOVO:bvr1HCN3?WW:*:svnLENOVO:pn80SG:*
+ ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, 1
+
 # IdeaPad Miix 320, different batches use a different sensor
 sensor:modalias:acpi:*BOSC0200*:dmi:*:svnLENOVO*:pn80XF:*
 sensor:modalias:acpi:SMO8840*:dmi:*:svnLENOVO:pn80XF:pvrLenovoMIIX320:*
